lib/src/core/component_config.dart

```diff
@@ -11,6 +12,7 @@ enum ComponentType {
   button('button'),
   textField('textField'),
   card('card'),
+  dropDown('dropdownButton'),
```

Comment:
Please change these to `text.field` and `dropdown.button`, dot separated, helps in making the type distinct than other values.

Reply:
Done